src/pages/Timetable/components/LectureList/index.tsx

```diff
@@ -107,7 +109,7 @@ function LectureList() {
   } = useSearch();
   const semester = useSemester();
   const { updateSemester } = useSemesterAction();
-  updateSemester(semesterParams || '20241');
+  updateSemester(semesterParams || '20242');
```

Comment:
상수로 빼도 괜찮을 거 같아요! 가장 최근 학기라는 뜻으로 `mostRecentSemester`도 괜찮은 것 같아요!
```suggestion
  updateSemester(semesterParams || mostRecentSemester);
```

Reply:
오! 가독성이 더 좋아지겠군요! 반영했습니다! 3c780299c2ff89468101b66fd87f10b985fdfad5